backend/src/api/nominatim.py

```diff
@@ -0,0 +1,25 @@
+import requests
+from typing import Tuple
+
+nominatim_endpoint = "https://nominatim.openstreetmap.org"
+address_query = "/search/{0}?format=json"
+
+
+def get_lat_long(address: str) -> Tuple[float, float]:
+    response = requests.get(nominatim_endpoint + address_query.format(address))
+
+    if response.status_code != 200:
+        # TODO: What's an appropriate return type if the request errors?
```

Comment:
Nothing much we can do if it errors. We should establish where in the code is responsible for catching errors such as inability to find a request. Either we return a None type or something similar, or we throw something such as a ConnectionError, and we do a try catch in HTTPServer, where if a ConnectionError is caught then we return to frontend a 500 or something similar